pages/webview/content.py

```diff
@@ -31,6 +32,10 @@ class Content(Page):
                           ' button[type="submit"][data-l10n-id="textbook-view-btn-get-this-book"]')
     )
     _ncy_locator = (By.CLASS_NAME, 'not-converted-yet')
+    _book_info_title_locator = (By.CSS_SELECTOR, '.booksContaining > div > span')
+    _go_to_book_link_locator = (By.CSS_SELECTOR, 'li:nth-child(3) > div > a')
+    _left_nav_book_title_locator = (By.CSS_SELECTOR, 'div.booksContaining > ul > li:nth-child(1) '
+                                                     '> div > a > b')
```

Comment:
In `pages/content_page.py` there is a Region called Book that represents a book in the book list. There is also a property on the page called `ContentPage.books_containing.books_list` where these elements should probably live.